### PR TITLE
Make changes to first set of docs

### PR DESCRIPTION
**Pull request by khorne3**
https://auth0-docs-content-pr-4900.herokuapp.com/docs//api-auth/dynamic-client-registration

https://auth0-docs-content-pr-4900.herokuapp.com/docs/migrations

https://auth0-docs-content-pr-4900.herokuapp.com/docs/support

### CODE REVIEW

File: articles/support/index.md [hunk 1]
```diff
@@ -59,19 +59,19 @@ Auth0 offers the following support channels.
...
+In addition to the Auth0 Community, paid subscribers can create a private ticket via [Support Center](${env.DOMAIN_URL_SUPPORT}). All tenant administrators will be able to view and add comments to Support Center tickets. Support Center can be accessed by clicking on the **Get Support** link on the [dashboard](${manage_url}).
 
 [Learn more about creating tickets with Support Center](/support/tickets)
 
 Critical Production issues should always be reported via the [Support Center](https://support.auth0.com/) for fastest response.
 
 #### Add Support-Only Users
 
-You can support-only to your Auth0 account. This will allow them to open, view, and comment on Suppoer Center tickets, receive relevant notifications, and see service details.  They do not, however, have full administration access to the Auth0 Dashboard.
+You can support-only users to your Auth0 tenant. This will allow them to open, view, and comment on Support Center tickets, receive relevant notifications, and see service details.  They do not, however, have full administration access to the Auth0 Dashboard.
```
Comment (mpaktiti) [inline articles/support/index.md]: Is there a verb missing here? Like `You can add support-only users...`

Comment (mpaktiti) [reply articles/support/index.md]: Add also links at Support Center and Dashboard

File: articles/api-auth/dynamic-client-registration.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ In order to access the [Management APIv2](/api/management/v2) endpoints from a t
 
 In this section we will see how you can enable the dynamic registration feature for your tenant.
 
-By default, the feature is disabled for all tenants. To change this, you have to update some account settings, promote the connections you will use with your dynamic clients to **domain connections**, and update your client's login page (if you use Lock).
+By default, the feature is disabled for all tenants. To change this, you have to update some tenant settings, promote the connections you will use with your dynamic clients to **domain connections**, and update your client's login page (if you use Lock).
 
 ::: panel-warning Security warning
```
Comment (mpaktiti) [inline articles/api-auth/dynamic-client-registration.md]: This is pretty short, we can use the `warning` UI element

File: articles/migrations/index.md [hunk 3]
```diff
@@ -127,11 +127,11 @@ Even if you are not using Lock, the vulnerable reset flow can be accessed direct
 | --- | --- | --- |
 | Medium | 2017-01-03 |  2017-03-01 |
 
-As part of Auth0's efforts to improve security and standards compliance, we will stop supporting account linking as part of the authorization callback (that is, accepting an `access_token` as part of the `/authorize` call as stated [here](/api/authentication?http#account-linking).
+As part of Auth0's efforts to improve security and standards compliance, we will stop supporting tenant linking as part of the authorization callback (that is, accepting an `access_token` as part of the `/authorize` call as stated [here](/api/authentication?http#account-linking).
```
Comment (mpaktiti) [inline articles/migrations/index.md]: I think you should undo this change, this refers to the account linking functionality https://auth0.com/docs/link-accounts

File: articles/migrations/index.md [hunk 4]
```diff
@@ -127,11 +127,11 @@ Even if you are not using Lock, the vulnerable reset flow can be accessed direct
...
 
 #### Am I affected by the change?
 
-If you received an email notification about it, then you are impacted by this change. As you work to update your applications to [use the Management API to link accounts](/api/management/v2#!/Users/post_identities), you can check if you are still impacted, by checking your tenant logs for warnings indicating _"Account linking via /authorize is being deprecated. Please refer to https://auth0.com/docs/link-accounts for supported ways to link an account."_. These entries will be logged if you are sending an `access_token` in your `/authorize` calls.
+If you received an email notification about it, then you are impacted by this change. As you work to update your applications to [use the Management API to link tenants](/api/management/v2#!/Users/post_identities), you can check if you are still impacted, by checking your tenant logs for warnings indicating _"Account linking via /authorize is being deprecated. Please refer to https://auth0.com/docs/link-accounts for supported ways to link a tenant."_. These entries will be logged if you are sending an `access_token` in your `/authorize` calls.
```
Comment (mpaktiti) [inline articles/migrations/index.md]: This too refers to account linking (as in the linking of user accounts from various identity providers)

File: articles/migrations/index.md [hunk 5]
```diff
@@ -208,7 +208,7 @@ You are affected by the change only if you are using the Identity Provider acces
 For more information on how to obtain an access token, see: [Call an Identity Provider API](/what-to-do-once-the-user-is-logged-in/calling-an-external-idp-api) and [Identity Provider Access Token](/tokens/idp).
 
 ::: note
-If your account was created after the change, this update will be applied automatically.
+If your tenant was created after the change, this update will be applied automatically.
```
Comment (mpaktiti) [inline articles/migrations/index.md:211]: Do you know this for sure? Are migrations applied based on when a tenant was created and not the account? (I'm not saying it's wrong, I'm just not sure)

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Some changes required

File: articles/support/index.md (hunk 1 above)
Comment (khorne3) [reply articles/support/index.md]: Bring your own verb! ;)

File: articles/migrations/index.md (hunk 5 above)
Comment (khorne3) [reply articles/migrations/index.md:211]: Yeah, I just reread the [notes](https://github.com/auth0/auth0-server/issues/2283). From what I understand, it has always been tenant, never account.
